locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
```

Comment:
This came from GitHub Flavored Markdown (GFM) https://guides.github.com/features/mastering-markdown/
They used "Task List", so I would like to propose "Todo List".
I think this is just a preference.